deel-2-flexbox-layout/opdracht-3/styles/styles.css

```diff
@@ -1 +1,38 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+body {
+    display: flex;
+    flex-direction: row;
+    min-height: 100vh;
+}
+
+nav {
+    flex-basis: 20vw;
+}
+
+div {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100vw;
+}
+
+header , footer {
+    flex-basis: 20vh;
+}
+
+main {
+    display: flex;
+    flex-direction: row;
+    flex-basis: 60vh;
+}
+
+article {
+    flex-basis: 60vw;
```

Comment:
Deze flex basis mocht volgens de uitwerkingen Novi met auto ingevuld, hoewel het in de opmaak niet echt uitmaakt geloof ik.